src/main/scala/com/microsoft/hyperspace/actions/CreateActionBase.scala

```diff
@@ -119,6 +120,37 @@ private[actions] abstract class CreateActionBase(dataManager: IndexDataManager)
           dataSchema.json,
           fileFormatName,
           opts)
+      case LogicalRelation(
+          HadoopFsRelation(location: TahoeLogFileIndex, _, dataSchema, _, fileFormat, options),
+          _,
+          _,
+          _) =>
+        val files = location
+          .getSnapshot(false)
+          .filesForScan(Seq(), Seq(), false)
+          .files
```

Comment:
Can we use `matchingFiles` or not?

Can you add comment why `Seq()` is fine for partition/data filters?

Reply:
`matchingFiles` seems to be used when there are additional data filters or partition filters. So I think we don't need to use it for now. 

And I added partitionFilter.